keras/src/utils/jax_layer.py

```diff
@@ -342,8 +442,15 @@ def _get_init_rng(self):
         Returns:
             a JAX `PRNGKey` or structure of `PRNGKey`s that will be passed as
             the `rng` argument of `init_fn`.
+
```

Comment:
Please update docstring of both `_get_init_rng` and `_get_call_rng`. They now returns an array of size 2, not a PRNGKey.